Resources/config/routing.yml

```diff
@@ -10,4 +10,4 @@ netgen_tags_admin_redirects:
 
 netgen_tags_admin:
     resource: "routing/admin.yml"
-    prefix: /tags/admin
+    prefix: /pjax/tags/admin
```

Comment:
I know, I mentioned that during the conference, this is something we'll improve, but for now that the only way. Hence why, the *WIP* in the title :)